### ISSUE TITLE
A11y: User Scalable

### ISSUE THREAD

**Issue by Link2Twenty** [thepracticaldev/dev.to#1214]
**Describe the bug**
Currently, users are not able to zoom in on content, if a user has poor eyesight they may not be able to use the sight without this.

https://github.com/thepracticaldev/dev.to/blob/1b60ee50091caf0d190e15f66b3771e0e9fb84cb/app/views/layouts/application.html.erb#L59
```HTML
<meta name="viewport" content="width=device-width, initial-scale=1.0">
```
**Expected behaviour**
A user should be able to zoom in.

**Additional context**
Another accessibility issue and another simple fix so again, a good first issues.

**Comment by Zhao-Andy** [comment]
I think this makes sense, any thoughts @benhalpern?

**Comment by benhalpern** [comment]
Finally getting to this, and seeing the PR.

I definitely agree with the principle. I feel like the native app experience on devices is one with fixed scalability, and there are accessibility features on the device which should support varying needs.

On the other hand, I don't unilaterally disagree, but I definitely want to consider this with a lot of care for the current and future features. 🙂

We can leave this open for discussion.

**Comment by maestromac** [comment]
not stale
